subsys/bluetooth/host/gatt.c

```diff
@@ -905,17 +905,35 @@ static u8_t get_service_handles(const struct bt_gatt_attr *attr,
 	return BT_GATT_ITER_CONTINUE;
 }
 
+static u16_t find_static_attr(const struct bt_gatt_attr *attr)
+{
+	const struct bt_gatt_service_static *static_svc;
+	u16_t handle;
+
+	for (static_svc = _bt_services_start, handle = 1;
+	     static_svc < _bt_services_end; static_svc++) {
+		for (int i = 0; i < static_svc->attr_count; i++, handle++) {
+			if (attr == &static_svc->attrs[i]) {
+				return handle;
+			}
+		}
+	}
+
+	return 0;
+}
+
 ssize_t bt_gatt_attr_read_included(struct bt_conn *conn,
 				   const struct bt_gatt_attr *attr,
 				   void *buf, u16_t len, u16_t offset)
 {
 	struct bt_gatt_attr *incl = attr->user_data;
+	u16_t handle = incl->handle ? : find_static_attr(incl);
```

Comment:
Aren't there more API functions that need this check?
`bt_gatt_attr_read_chrc` for example?

Reply:
Yep, that seems broken as well which is quite surprising given it seems to works with the like of nrfconnet and bluetoothctl, anyway these should be fixed now.